src/app/helper-families/helper-families.component.ts

```diff
@@ -145,7 +146,7 @@ export class HelperFamiliesComponent implements OnInit {
         fd.floor,
         fd.city],
       from: fd,
-      where: () => [fd.deliverStatus.isEqualTo(DeliveryStatus.ReadyForDelivery).and(fd.courier.isEqualTo('')).and(fd.quantity.isLessThan(settings.maxDeliverisQuantityToShowForIndependent))]
+      where: () => [fd.deliverStatus.isEqualTo(DeliveryStatus.ReadyForDelivery).and(fd.courier.isEqualTo('')).and(fd.quantity.isLessOrEqualTo(settings.MaxItemsQuantityInDeliveryThatAnIndependentVolunteerCanSee)).and(fd.familySource.isDifferentFrom('0b9e0645-206a-457c-8785-97163073366d'))]
```

Comment:
what is this excluded family?

Reply:
`donationType = new StringColumn("סוג תרומה", {
    dataControlSettings: () => ({
      valueList: [{ id: 'ac52f4b0-6896-4ae3-8cc0-18ed17136e38', caption: 'תרומה פרטית' },
      { id: '0b9e0645-206a-457c-8785-97163073366d', caption: 'תרומת בית עסק' }]
    })
  })`

I took it from this code